sql/codegen.go

```diff
@@ -324,52 +324,70 @@ print("Training set accuracy: {accuracy:0.5f}".format(**eval_result))
 print("Done training")
 {{- else}}
 
-def eval_input_fn(features, batch_size):
-	dataset = tf.data.Dataset.from_tensor_slices(dict(features))
-	return dataset.batch(batch_size)
-
-field_names, columns = execute(driver, conn, """{{.StandardSelect}}""")
```

Comment:
Since the `execute` is removed, we may remove it from the [import line](https://github.com/sql-machine-learning/sqlflow/pull/510/files#diff-a9c90abc2107d6cd0f30e093777900bfR200) either. 
What is more, I would suggest move the `execute` from `db.py` and `maxcompute.py` to `db_test.py`.

Reply:
Done.